client/packages/invoices/src/Prescriptions/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -133,7 +134,9 @@ export const StatusChangeButton = () => {
     useStatusChangeButton();
   const isDisabled = usePrescription.utils.isDisabled();
   const t = useTranslation();
-  const noLines = lines?.totalCount === 0;
+  const noLines =
+    lines?.totalCount === 0 ||
+    lines?.nodes?.every(l => l.type === InvoiceLineNodeType.UnallocatedStock);
```

Comment:
This button also disabled rather than toasting... wanting to be consistent?

Reply:
Opps thought I removed them thanks Lache :D

Reply:
Hehe oops, gotta add the toast here now 😁

Reply:
Nope, my bad should have triple checked